src/dev/jest/setup_after_env.integration.js

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
__question:__ do you think it makes to have this file in ` src/dev/jest/setup/` with other setup-like files? Not a big deal though.